media/com_templates/js/admin-template-toggle-switch.es6.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * @copyright  Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+(() => {
+  'use strict';
+
+  const showDiffChangedOff = function showDiffChangedOff() {
```

Comment:
Can we use arrow function in es6 file ?
`const showCoreChangedOff = () => { .. }`